### ISSUE TITLE
Incorrect spaces count of East Asian characters in console.

### ISSUE THREAD

**Issue by ycs77** [laravel/framework#52878]
### Laravel Version

10.48.22

### PHP Version

8.1.29

### Description

I am using the alert and two columns components of Laravel, If I enter the English it will work fine:

![termwind-1](https://github.com/nunomaduro/termwind/assets/38133356/4b909b74-26bb-4d72-a9bf-a7a3077c5ab3)

But if I enter East Asian characters such as full-width characters (such as Chinese or Japanese) it will show incorrect spaces:

![termwind-2](https://github.com/nunomaduro/termwind/assets/38133356/f8dbf666-d7c1-48fe-b129-5a6e2a8a80ca)
![termwind-3](https://github.com/nunomaduro/termwind/assets/38133356/d3681ebc-a392-465f-92e2-7d887b4871f1)

> linked issue: https://github.com/nunomaduro/termwind/issues/185
>
> linked PR: https://github.com/nunomaduro/termwind/pull/186

### Steps To Reproduce

1. initial a new Laravel project
2. create a command: `php artisan make:command HelloWorld`
3. add under code in `handle()`

```php
public function handle()
{
    $helloWorld = 'Hello World!';
    // $helloWorld = '你好 世界';
    // $helloWorld = 'ハロー ワールド';

    $this->components->alert($helloWorld);

    $this->newLine();

    $this->components->twoColumnDetail('Termwind', $helloWorld);
}
```

4. run `php artisan app:hello-world`
5. change the different language of the `$helloWorld` to see the view

**Comment by crynobone** [comment]
It seems PR to termwind should fix this issue.

**Comment by ycs77** [comment]
Hi @taylorotwell,

Could you help me advance to merge the Termwind PR (https://github.com/nunomaduro/termwind/pull/186) that addresses the CJK character spacing issue reported in this Laravel issue?

While this issue has been closed here, the corresponding pull request in the Termwind repository has been pending for some time without being merged. This issue affects the user experience for me and CJK language users when working with Laravel's console components and Pest result print.

I would be very grateful if you could help advance to merging of this PR or provide some guidance on any additional steps needed to move it forward.

Thank you very much for your time and consideration, and for all your incredible work on Laravel.

---

Linked PR: https://github.com/nunomaduro/termwind/pull/186
Linked issues: https://github.com/nunomaduro/termwind/issues/185, https://github.com/pestphp/pest/issues/1504

**Comment by ycs77** [comment]
Fixed in termwind v2.3.2

**Comment by ycs77** [comment]
Screenshots after fixed:

<img width="1658" height="820" alt="Image" src="https://github.com/user-attachments/assets/c318ca21-4d37-414f-b545-50b7bcb8780b" />
